package.json

```diff
@@ -34,7 +34,7 @@
     "coverage-publish": "aegir coverage publish"
   },
   "dependencies": {
-    "base-x": "^3.0.8",
+    "base-x": "achingbrain/base-x#feature/return-uint8arrays",
```

Comment:
```suggestion
    "base-x": "@multiformats/base-x",
```

Reply:
Needs to be “@multiformats/base-x”: “...version”...